logpolicy/logpolicy.go

```diff
@@ -308,8 +309,8 @@ func tryFixLogStateLocation(dir, cmdname string) {
 }
 
 // New returns a new log policy (a logger and its instance ID) for a
-// given collection name.
-func New(collection string) *Policy {
+// given collection name logging to a given target.
```

Comment:
explain what a target is and the default value.

but positional args of the same type (two strings) back to back are generally bad, especially when they're optional. I'd normally put it as a field on Policy instead (like SetVerbosityLevel), and have Policy lazily do its setup on first use, but that might be too invasive at this point for how weird this "constructor" is... it returns a thing but the whole point of the New func isn't to return a Policy but to do a bunch of side effects.

So I guess just document this a bunch for now.